keyboards/mechmini/v2/keymaps/wsturgiss/keymap.c

```diff
@@ -0,0 +1,108 @@
+#include QMK_KEYBOARD_H
+#include "config.h"
+#include "../../../../../quantum/rgblight_list.h"
```

Comment:
This line is not actually needed, and should be removed. 

It's automatically included at build time.

Reply:
@drashna's comment still needs attention, but everything else here looks good to me.